osu.Game/Rulesets/Mods/ModFlashlight.cs

```diff
@@ -86,6 +86,7 @@ public virtual void ApplyToDrawableRuleset(DrawableRuleset<T> drawableRuleset)
             flashlight.Depth = float.MinValue;
 
             flashlight.Combo.BindTo(Combo);
+            flashlight.GetPlayfieldScale = () => drawableRuleset.Playfield.Scale;
```

Comment:
I considered storing the entire `DrawableRuleset` like `InputBlockingMod` does, but ran into issue with generics and found this to be the cleanest solution instead. Any suggestions?

The scale cannot be retrieved at this point as BarrelRoll is not guarenteed to apply it's scale change by the time this is called.